src/stores/widgets/ElementWidgetActions.ts

```diff
@@ -21,8 +21,6 @@ export enum ElementWidgetActions {
     JoinCall = "io.element.join",
     HangupCall = "im.vector.hangup",
     CallParticipants = "io.element.participants",
-    MuteAudio = "io.element.mute_audio",
-    UnmuteAudio = "io.element.unmute_audio",
```

Comment:
These were just unused?

Reply:
yes I could not find where they were used in EC, the react sdk and the widget api.

Reply:
They were used by Jitsi.

https://github.com/element-hq/element-web/blob/develop/src/vector/jitsi/index.ts#L197-L206

If they were stale then they should have been removed on that side first

Reply:
Looks like https://github.com/matrix-org/matrix-react-sdk/commit/1f64835fabe9f3471c368da5169d2805a800cde8 stopped the use of those enum values

Reply:
@robintown as the author of that commit do you have any additional context?

Reply:
Looks like it powered a set of quick controls in the left panel which were removed

![image](https://github.com/user-attachments/assets/2e9eabc7-2df8-42cd-b209-9e691767b97e)

Reply:
https://github.com/element-hq/element-web/pull/27858 should fix it